examples/nfct_test.rs

```diff
@@ -0,0 +1,170 @@
+#![no_std]
+
+extern crate alloc;
+extern crate lang_items;
+extern crate libtock_drivers;
+
+use core::fmt::Write;
+use libtock_core::result::CommandError;
+use libtock_drivers::console::Console;
+use libtock_drivers::nfc::NfcTag;
+use libtock_drivers::nfc::RecvOp;
+use libtock_drivers::result::FlexUnwrap;
+use libtock_drivers::result::TockError;
+use libtock_drivers::timer;
+use libtock_drivers::timer::Timer;
+use libtock_drivers::timer::Timestamp;
+
+#[allow(dead_code)]
+/// Helper function to write on console the received packet.
+fn print_rx_buffer(buf: &mut [u8], amount: usize) {
+    if amount < 1 || amount > buf.len() {
+        return;
+    }
+    let mut console = Console::new();
+    write!(console, "RX:").unwrap();
+    for byte in buf.iter().take(amount - 1) {
```

Comment:
For what you are trying to do here, there is [`split_last`](https://doc.rust-lang.org/std/primitive.slice.html#method.split_last). To use that, the slice length would need to match the `amount`. So maybe it makes sense to pass in the sub-slice of the correct length? Then this function wouldn't look so C-style.